### PR TITLE
Fix/sst 57

### PR DESCRIPTION
**Pull request by CaroBoud**
## Description
Deleted some unused missions

## Fixes
Closes jira mission cleanup sst-57

## How has this been tested ?
Behaviors have been opened, and some were tested in simulation to check if nothing was break.

## Checklist
- [ X ] My code follows the style guidelines of this project
- [ X ] I have performed a self-review of my own code
- [ X ] I have commented my code, particularly in hard-to-understand areas
- [ X ] I have made corresponding changes to the documentation
- [ X ] My changes generate no new warnings

### CODE REVIEW

File: sonia_flexbe_behaviors/manifest/search_square.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
 	<author>CS</author>
 	<date>July 12 2022</date>
 	<description>
-		Search mouvement in a zigzag patern in a square.
+		Search mouvement in a circle patern in a square.
```
Comment (S1lver-Owl) [inline sonia_flexbe_behaviors/manifest/search_square.xml]: That sentence makes no sense

File: sonia_flexbe_behaviors/src/sonia_flexbe_behaviors/search_square_sm.py [hunk 2]
```diff
@@ -24,7 +24,7 @@
 '''
 class search_squareSM(Behavior):
 	'''
-	Search mouvement in a zigzag patern in a square.
+	Search mouvement in a circle patern in a square.
```
Comment (S1lver-Owl) [inline sonia_flexbe_behaviors/src/sonia_flexbe_behaviors/search_square_sm.py]: Same as above, Grammar is weird

**Review by S1lver-Owl — COMMENTED** [review]
This and making sure that no other missions fail.
Other than that, LGTM!

File: sonia_flexbe_behaviors/manifest/search_square.xml [hunk 3]
```diff
@@ -7,7 +7,7 @@
 	<author>CS</author>
 	<date>July 12 2022</date>
 	<description>
-		Search mouvement in a zigzag patern in a square.
+		This state uses a circular search pattern in the form of an expanding square.
```
Comment (S1lver-Owl) [inline sonia_flexbe_behaviors/manifest/search_square.xml]: on a pas dis spiral?

File: sonia_flexbe_behaviors/src/sonia_flexbe_behaviors/search_square_sm.py [hunk 4]
```diff
@@ -24,7 +24,7 @@
 '''
 class search_squareSM(Behavior):
 	'''
-	Search mouvement in a zigzag patern in a square.
+	This state uses a circular search pattern in the form of an expanding square.
```
Comment (S1lver-Owl) [inline sonia_flexbe_behaviors/src/sonia_flexbe_behaviors/search_square_sm.py]: meme que l'autre message

**Review by S1lver-Owl — APPROVED** [review]
other than the comments, LGTM

File: sonia_flexbe_behaviors/manifest/search_square.xml (hunk 3 above)
Comment (CaroBoud) [reply sonia_flexbe_behaviors/manifest/search_square.xml]: Oui c'est vrai ! Je l'ai changé, merci :)

**Pull request merged**